src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RAZMER = 15;
```

Comment:
Use google translator)
```suggestion
    private static final int MAX_SIZE = 15;
```